### ISSUE TITLE
false positive for use of valid role=image

### ISSUE THREAD

**Issue by scottaohara** [dequelabs/axe-core#4656]
### Product

axe-core

### Product Version

4.10.2

### Latest Version

- [x] I have tested the issue with the latest version of the product

### Issue Description

per the following test
```
<div role=image>foo</div>
```

the checker reports a"ARIA roles used must conform to valid values".  `role=image`  is a [valid role](https://w3c.github.io/aria/#image) which is supported by all browsers ([wpt demonstrating role=image supported in all browsers](https://wpt.fyi/results/wai-aria/role/synonym-roles.html?label=pr_head&max-count=1&pr=42203).

ARIA in HTML is updating to indicate this is an allowed role for all the instances of where `role=img` was also allowed.

**Comment by straker** [comment]
Thanks for letting us know.

**Comment by WilcoFiers** [comment]
We're going to need to test how well supported this is. I'd wager since this only just want in it isn't there yet. We'd have to report this as an unsupported role though, not as an invalid role.

**Comment by scottaohara** [comment]
definitely do your own testing to verify.  but as mentioned, it's supported in all browsers as demonstrated by the passing WPT i linked to.  all screen readers treat it exactly the same as `role=img`, because the mapping are identical.

the only place where i'd imagine that maybe there wouldn't be support is with a user agent that doesn't use the accessibility tree, but instead still looks at just the dom.

**Comment by WilcoFiers** [comment]
Instead of implementing this throughout, I think we should add a synonym prop on our standards object, and have `getRole()` return the synonym by default.

```js
elm.role=image
getRole(elm) // img
getRole(elm, { noSynonym }) // image
```

That would then let us set presentation as a synonym of none as well, so that we can ditch the `['none', 'presentation'].includes(role)` calls we have all over the place.
